XpringKit/XRP/Model/SendXRPDetails.swift

```diff
@@ -0,0 +1,28 @@
+/// Describes the fine grained details for sending XRP. The destination field may be a PayID, XAddress,
+/// or other type of address. Handling of the given destination type is the responsibility of the client.
+public struct SendXRPDetails {
+
+  /// The amount of XRP, in drops, to send.
+  public let amount: UInt64
+
+  /// The receiving address.
+  public let destination: Address
+
+  /// The sending wallet.
+  public let sender: Wallet
+
+  /// A list of memos to attach to the payment transaction constructed from these details.
+  public let memosList: [XRPMemo]?
+
+  public init(
```

Comment:
you can omit this constructor and let Swift generate it for you.

Reply:
I can't in this case actually because I need `SendXRPDetails` to have a publicly accessible constructor, and according to this I have to implement that myself: https://stackoverflow.com/questions/26224693/how-can-i-make-the-memberwise-initialiser-public-by-default-for-structs-in-swi